torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -878,6 +898,8 @@ def is_all_inputs_connected_to_quantized_op(input_nodes):
                 if not is_all_inputs_connected_to_quantized_op(input_nodes_to_check):
                     return
                 self._annotate_cat(node, quantization_config)
+            elif node.target is torch.ops.aten.embedding_bag.padding_idx:
```

Comment:
`_annotation_propagation_quantizable_pattern` is used to annotate operators like `maxpool2d`, `cat` which will depend on previous nodes to decide whether we should insert observer for it. But here, the recipe for `embedding_bag` is insert observer at weight no matter of previous node.

- Can we double check in IPEX whether it's a desired recipe?
- If so, I think the implementation of recipe for `embedding_bag` shouldn't in `_annotation_propagation_quantizable_pattern` but somewhere like the conv/linear.